config/jobs/etcd/etcd-presubmits.yaml

```diff
@@ -14,9 +14,12 @@ presubmits:
       - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20240515-17c6d50e24-master
         command:
         - runner.sh
-        args:
-        - make
-        - build
+        - |
+          targets=('linux-amd64' 'linux-arm64' 'linux-arm' 'linux-ppc64le' 'linux-s390x' 'linux-386' 'darwin-amd64' 'darwin-arm64' 'windows-amd64')
```

Comment:
I _highly_ suggest that you move this into a make target in the repo so the list can be changed and tested in a PR to that repo instead of here.

I do not recommend putting complex logic inline in the CI definition, especially if you already have a makefile and there's nothing CI environment specific about this logic.

(also with a list this long you may need to consider parallelism)

Reply:
ah, this is https://github.com/kubernetes/test-infra/pull/32672#discussion_r1616735510 🙃

Reply:
Thanks @BenTheElder , I have opened a PR to address it. Once that PR is merged, I will update this prowjob.